### PR TITLE
Adds volume-profile label in openshift-csi manifest

### PR DESCRIPTION
**Pull request by luhi-DT**
## Description

Currently the CSI manifests on OCP are missing the "csi-ephemeral-volume-profile" label. The check is wrong to add it, when the manifests are generated, as the check expects "partial" to be false, but it actually is "csi" in case of openshift-csi.yaml.

## How can this be tested?

make manifests -> should generate an openshift-csi.yaml that has the CSIDriver resource and the label on it

## Checklist

- [ ] Unit tests have been updated/added
- [x] PR is labeled accordingly with a single label
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: config/helm/chart/default/templates/Common/csi/csidriver.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ kind: CSIDriver
 metadata:
   name: csi.oneagent.dynatrace.com
   labels:
-    {{- if (eq (include "dynatrace-operator.openshiftOrOlm" .) "true") }}
+    {{- if eq (include "dynatrace-operator.platform" .) "openshift" }}
```
Comment (andriisoldatenko) [inline config/helm/chart/default/templates/Common/csi/csidriver.yaml:21]: can you please update or add new helm test?

Comment (aorcholski) [reply config/helm/chart/default/templates/Common/csi/csidriver.yaml:21]: The code depends on 3 variables 
- `platform`
- needCSI (`or`): 
  - `csidriver.enabled`
  - `partial`

The test should also set all of them and cover at least 2 basic cases
```
- it: should contain correct labels for openshift with CSI enabled
    set:
      platform: openshift
      csidriver.enabled: true
      partial: false

  - it: should contain correct labels for openshift manifest
    set:
      platform: openshift
      csidriver.enabled: false
      partial: csi
```

Comment (aorcholski) [reply config/helm/chart/default/templates/Common/csi/csidriver.yaml:21]: please add `partial`  to the first case 

```
  - it: should contain correct labels for openshift with CSI enabled
    set:
      platform: openshift
      csidriver.enabled: true
      partial:false                     <----
    asserts:
```

Comment (aorcholski) [reply config/helm/chart/default/templates/Common/csi/csidriver.yaml:21]: thanks!

**Review by aorcholski — APPROVED** [review]

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**